recirq/time_crystals/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from recirq.time_crystals.dtctask import *
```

Comment:
try to avoid `*` imports. this will pull in everything into `recirq.time_crystals` including `os`, `datetime`, etc